### DISCUSSION TITLE
Add a delay between taps, and allow user to choose how many taps

### DISCUSSION THREAD

**Discussion by joshuazzz1** [masterking32/MasterHamsterKombatBot#169] [Ideas]
Make the number of taps configurable and make the script choose a random number between those, instead of doing all of the taps at the same time, so it doesn't raise suspicions, also add a delay in-between them which will be configurable too, for example in the config

"tap_count_min": 421,
"tap_count_max": 5378,
"tap_delay_min": 5, # in seconds
"tap_delay_max": 10,

**Comment by masterking32** [comment]
https://github.com/masterking32/MasterHamsterKombatBot/issues/51
